### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by skatun**
Added some background info about sensors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,15 @@ After data made the hop to the MQTT broker it can be used by home automation sof
 
 The program can be executed in **daemon mode** to run continuously in the background, e.g., as a systemd service.
 
+## About Mi Flora
+* Support for [Xiaomi Mi Flora sensors](https://xiaomi-mi.com/sockets-and-sensors/xiaomi-huahuacaocao-flower-care-smart-monitor) ([e.g. 12-17€](https://www.aliexpress.com/wholesale?SearchText=xiaomi+mi+flora+plant+sensor))
+* The sensor are meant to keep your plants alive by monitoring soil moisture, soil conductivity and light conditions
+* The sensors uses BLE and broadcast it every 30s
```
Comment (ThomDietrich) [inline README.md]: what is "it"?

Comment (ThomDietrich) [reply README.md]: @skatun ping ;)

**Comment by ThomDietrich** [comment]
@skatun ping

File: README.md (hunk 1 above)
Comment (skatun) [reply README.md]: Hi, I just added some information about update frequency, we should also add something about battery life..

**Comment by ThomDietrich** [comment]
Thanks!!

**Pull request merged**